### PR TITLE
Added a 'is_view' and 'Table View' indicator on the table details page

### PR DESCRIPTION
**Pull request by danwom**

### CODE REVIEW

**Comment by danwom** [comment]
CC: @youngyjd

**Comment by youngyjd** [comment]
💯 awesome!

File: amundsen_application/static/js/components/common/Flag/styles.scss [hunk 1]
```diff
@@ -1,6 +1,7 @@
 @import 'variables';
 
 .flag {
-  margin-right: 4px;
-  margin-left: 4px;
+  display: inline-block;
```
Comment (ttannis) [inline amundsen_application/static/js/components/common/Flag/styles.scss]: I wonder if these changes affect how the `Flag` was intended to be laid out on the `ProfilePage`. Can we either:
1. Verify the use of `Flag` on `ProfilePage` still renders according to the design with these styles OR
2. Override these styles for `Flag` on `TableDetail` page only, and then revisit #1 later to consolidate styles if we can.

Comment (danwom) [reply amundsen_application/static/js/components/common/Flag/styles.scss]: The profile page currently overrides these styles and still renders correctly.

**Pull request merged**